deployment/deploy/upgrade.sh

```diff
@@ -59,13 +59,22 @@ getId() {
 update() {
     TYPE=$1
     ID=`docker ps -f "name=service-$TYPE" --format "{{.ID}}"`
+    if [ -f DOMAIN_NAME.$TYPE ]; then
+        LOCAL_DN=`cat DOMAIN_NAME.$TYPE | tr -d '\n'`
+    else
+        LOCAL_DN=$DOMAIN_NAME
+    fi
     if [ "$ID" != "" ]; then
         # Make sure everything is up to date
         docker exec -it $ID apt-get update
         docker exec -it $ID apt-get upgrade -y
         # Make sure that all files are there if needed
         docker cp ../docker/fs/. $ID:/tmp/fs/
-        docker exec -it $ID bash -c "echo $TYPE.$DOMAIN_NAME > /etc/hostname"
+        if [ -d ../docker/fs.$TYPE ]; then
+            cp ../docker/fs.$TYPE/. $ID:/tmp/fs/
```

Comment:
`docker cp ...`?

Reply:
Thank you! This is why we have peer review. Should be fixed now.